tests/integration/framework/process/helm/helm.go

```diff
@@ -31,7 +33,17 @@ type Helm struct {
 func New(t *testing.T, fopts ...OptionFunc) *Helm {
 	t.Helper()
 
-	var opts options
+	opts := options{
+		// Since helmtemplate self exists, expect it to always run without error,
```

Comment:
is this true? a bad helm template would make it fail probably: 
```
helm template charts/dapr
Error: template: dapr/charts/dapr_scheduler/templates/dapr_scheduler_statefulset.yaml:144:13: executing "dapr/charts/dapr_scheduler/templates/dapr_scheduler_statefulset.yaml" at <include "dapr_scheduler.etcdclienthttpports" .>: error calling include: template: dapr/charts/dapr_scheduler/templates/_helpers.tpl:70:4: executing "dapr_scheduler.etcdclienthttpports" at <append .Values.ports.etcdHTTPClientPortadfads 0>: error calling append: runtime error: invalid memory address or nil pointer dereference

Use --debug flag to render out invalid YAML
(base) filinto@Filintos-MacBook-Pro dapr % echo $?
1
```

Reply:
Windows will exit error when interrupting a long running process. By default `exec` handle this as all processes have been long running until now. Since `helmtemplate` exits success on it's own, we overwrite this default to always expect non-error exits- even on windows.